firebase-perf/src/main/java/com/google/firebase/perf/injection/modules/GaugeModule.java

```diff
@@ -0,0 +1,48 @@
+// Copyright 2021 Google LLC
```

Comment:
Usually it's a good idea to have the modules together with the class it applies to.

Reply:
Make sense! Done